Digital Twin/procedures/cb.getTheme.vail

```diff
@@ -0,0 +1,27 @@
+PROCEDURE cb.getTheme()
```

Comment:
The theme format needs documentation minimally, and ideally we would get rid of it and only use the native client theming capacity.